tests/template_tests/test_context.py

```diff
@@ -43,6 +43,20 @@ def test_update_context_manager(self):
             self.assertEqual(c['a'], 3)
         self.assertEqual(c['a'], 1)
 
+    def test_push_context_manager_with_context_object(self):
+        c = Context({"a": 1})
```

Comment:
I guess you copied form existing tests, but it would be nice to be consistent in use of quotes (I'd go with single here). Also I think we could omit the blank lines.